### DISCUSSION TITLE
Route Type parameter in routes API Call returns parameter error

### DISCUSSION THREAD

**Discussion by dinwun450** [transitland/transitland#435] [Q&A] [answered]
Before encountering this error, I could filter routes based on the type of transportation using the `route_type` parameter from the route querying documentation. But when I tried to query the specific routes using the `route_type` parameter now, it returned a "parameter error". Was there a bug?

**Comment by irees** [comment, marked as answer]
hi @dinwun450 

We looked into it, and fixed the issue. It was a regression on the REST side of the api, related to new parameter `route_types` for allowing multiple values (e.g. `route_types=1,2,3`). Thank you for alerting us to the problem.
